pom.xml

```diff
@@ -26,9 +26,36 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>com.uber.tchannel</groupId>
-    <artifactId>tchannel</artifactId>
+    <artifactId>tchannel-java</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
```

Comment:
so, my understanding is that we should/could continually update SNAPSHOT builds i.e. un-versioned nightlies, and then release with traditional SemVer semantics e.g. Major.Minor.Patch. 

What do you think / propose as an alternative?